### PR TITLE
Additions

### PR DESCRIPTION
**Pull request by ayu-ayushisood**
• Added Bootstrap
• Fixed Homepage styling
• Added Navbar and its links
• Added theme related Font

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ import Home from './Screens/home'
 import Experience from './Screens/experience';
 import Work from './Screens/work';
 import Skills from './Screens/skills';
+import 'bootstrap/dist/css/bootstrap.min.css';
```
Comment (karanasthana) [inline src/App.js]: why is it being imported like this ?

File: src/App.css [hunk 2]
```diff
@@ -1,12 +1,24 @@
...
+    --themeBlue: #3C465F;
+    --themeYellow: #FFB038;
+    --themeRed: #AB4C4C;
     --fullwidth: 100%;
 }
 .main_app_container {
     height: 100%;
     width: 100%;
     position: absolute;
+    font-family: 'Maven-pro', 'Maven-pro-bold', Arial, sans-serif;
```
Comment (karanasthana) [inline src/App.css]: why do you have `Maven-pro-bold` here as the fallback for default font? 

I don't think you'll ever want main_app_container to be bold for all fonts there

File: src/assets/fonts/Maven_Pro/README.txt [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+Maven Pro is a variable font with this axis:
+  wght
+
+This means all the styles are contained in a single file:
+  MavenPro-VariableFont_wght.ttf
+
+If your app fully supports variable fonts, you can now pick intermediate styles
+that aren’t available as static fonts. Not all apps support variable fonts, and
+in those cases you can use the static font files for Maven Pro:
+  static/MavenPro-Regular.ttf
```
Comment (karanasthana) [inline src/assets/fonts/Maven_Pro/README.txt:15]: You're not using these fonts I believe. (Except Maven-Pro and Maven-Pro-Bold)

File: src/assets/fonts/Maven_Pro/OFL.txt [hunk 4]
```diff
@@ -0,0 +1,93 @@
+Copyright 2011 The Maven Pro Project Authors (http://www.vissol.co.uk/mavenpro/), with Reserved Font Name "Maven Pro".
```
Comment (karanasthana) [inline src/assets/fonts/Maven_Pro/OFL.txt:1]: is this file required ?

File: src/Screens/landing/style.css [hunk 5]
```diff
@@ -1,52 +1,60 @@
...
+    align-items: center;
 }
 
 .top-section {
-    background-color: var(--yellow);
+    /* background-color: var(--themeYellow); */
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    height: 22em;
```
Comment (karanasthana) [inline src/Screens/landing/style.css:21]: Are you sure you want to use height?

Can there be any alternative? 

Maybe, if height, use vh ?

File: src/Screens/landing/index.js [hunk 6]
```diff
@@ -1,31 +1,69 @@
...
+                        <div className="headline">
+                            <h2>Frontend Developer | Mobile app Developer <br />{`</>`}</h2>
+                        </div>
+                    </div>
+                    {/* <div className="code-symbol">
+                        <h2>{`</>`}</h2>
+                    </div> */}
                 </div>
+            </div>
+            <div className="bottom-section">
```
Comment (karanasthana) [inline src/Screens/landing/index.js:56]: use a better name probably...
`bottom-section-top-fold` OR `top-container-profile-img-section`

File: src/Screens/landing/style.css [hunk 7]
```diff
@@ -55,13 +63,38 @@
...
+    background-color: var(--themeBlue);
     color: white;
     width: 80%;
     height: fit-content;
-    position: relative;
     padding: 0 1em;
     font-size: 2em;
-    top: 28%;
+    text-align: center;
+    line-height: normal;
```
Comment (karanasthana) [inline src/Screens/landing/style.css:73]: is `line-height: normal` required ?

File: src/Screens/landing/style.css [hunk 8]
```diff
@@ -1,52 +1,60 @@
...
     background-color: white;
     height: 240px;
     width: 240px;
-    position: absolute;
-    top: 32%;
     border-radius: 120px;
     display: flex;
     justify-content: center;
     align-items: center;
+    margin-top: -114px;
```
Comment (karanasthana) [inline src/Screens/landing/style.css]: have you tested this on responsive ?

I think it is better if we can use `em` or `rem` for this.

File: src/Screens/landing/style.css [hunk 9]
```diff
@@ -1,52 +1,60 @@
...
-    top: 23.5%;
-    color: var(--blue);
+    margin-top: 1.5em;
+    color: var(--themeBlue);
+    text-align: center;
 }
-
 .image-container {
     background-color: white;
     height: 240px;
```
Comment (karanasthana) [inline src/Screens/landing/style.css:51]: have you tested this on responsive ?

I think it is better if we can use `em` or `rem` for this.

Or decrease height and width in media screens

**Review by karanasthana — CHANGES_REQUESTED** [review]

File: src/Screens/landing/style.css (hunk 7 above)
Comment (ayu-ayushisood) [reply src/Screens/landing/style.css:73]: yes

File: src/App.css (hunk 2 above)
Comment (ayu-ayushisood) [reply src/App.css]: correct, removed!

File: src/App.js (hunk 1 above)
Comment (ayu-ayushisood) [reply src/App.js]: moved to root file

File: src/assets/fonts/Maven_Pro/README.txt (hunk 3 above)
Comment (ayu-ayushisood) [reply src/assets/fonts/Maven_Pro/README.txt:15]: yes but these files are part of Font Package (default)

File: src/assets/fonts/Maven_Pro/OFL.txt (hunk 4 above)
Comment (ayu-ayushisood) [reply src/assets/fonts/Maven_Pro/OFL.txt:1]: no but these files are part of Font Package (default)
can be removed

File: src/Screens/landing/style.css (hunk 5 above)
Comment (ayu-ayushisood) [reply src/Screens/landing/style.css:21]: height is needed and vh will break the entire layout in this case.

File: src/Screens/landing/index.js (hunk 6 above)
Comment (ayu-ayushisood) [reply src/Screens/landing/index.js:56]: bottom-section is correct according to the layout.
Nav then top-section then bottom-section. So.

File: src/Screens/landing/style.css (hunk 7 above)
Comment (ayu-ayushisood) [reply src/Screens/landing/style.css:73]: by default it was 1.5 form the body

File: src/Screens/landing/style.css (hunk 8 above)
Comment (ayu-ayushisood) [reply src/Screens/landing/style.css]: changed to em, it works

File: src/Screens/landing/style.css (hunk 9 above)
Comment (ayu-ayushisood) [reply src/Screens/landing/style.css:51]: yes, it will be decreased in media queries.
